src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+
+  if (days < 3) {
+    return perDay * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return perDay * days - 20;
+  }
+
+  if (days >= 7) {
+    return perDay * days - 50;
+  }
```

Comment:
would be great to create variables for numbers
check [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)